mappings/net/minecraft/world/World.mapping

```diff
@@ -47,8 +47,18 @@ CLASS net/minecraft/class_18 net/minecraft/world/World
 	METHOD method_180 setState (Ljava/lang/String;Lnet/minecraft/class_58;)V
 		ARG 1 id
 		ARG 2 state
+	METHOD method_189 findPathToLocation (Lnet/minecraft/class_57;IIIF)Lnet/minecraft/class_61;
```

Comment:
this should probably be `findPathToPosition` for same reason as in PathNodeNavigator, altho [findPathTo](https://maven.fabricmc.net/docs/yarn-1.20.6+build.3/net/minecraft/entity/ai/pathing/EntityNavigation.html#findPathTo(net.minecraft.util.math.BlockPos,int)) might be better